cypress/integration/jsTaskParseListDOM.spec.js

```diff
@@ -14,15 +14,14 @@ describe('Parse list app ', () => {
   });
 
   it('should be sorted by salary DSC', () => {
-
     cy.get('ul > li').then(($ul) => {
-      const list = [...$ul].map(
-          (row) => row.dataset.salary.replace('$', '').replace(',', ''));
+      const list = [...$ul].map((row) =>
+        row.dataset.salary.replace('$', '').replace(',', ''),
```

Comment:
Chaining 'replace' calls can be error-prone and hard to read. Consider using a single replace with a regular expression to clean up the salary strings.